### ISSUE TITLE
SC2046 false positives without workarounds

### ISSUE THREAD

**Issue by ssbarnea** [koalaman/shellcheck#967]
#### For bugs
- SC2046: 
- 0.4.6
- [ ] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### Here's a snippet or screenshot that shows the problem:

```#!/bin/bash
wait $(jobs -p)
```

#### Here's what I wanted or expected to see:
No warnings or one warning that tells me how to disable to rule for current line.

**Comment by koalaman** [comment]
See the man page or [the wiki Ignore page](https://github.com/koalaman/shellcheck/wiki/Ignore) for how to generally ignore errors per statement.
